Material.Blazor/Foundation/ComponentFoundation.cs

```diff
@@ -56,6 +56,13 @@ public abstract class ComponentFoundation : ComponentBase, IAsyncDisposable
         }
 
 
+        [Parameter] public string id { get; set; }
+        [Parameter] public string @class { get; set; }
```

Comment:
I used lower-case identifiers here on purpose. This matches what most people are probably doing, even if the html standard allows things to be upper case too.